### DISCUSSION TITLE
Plugin to chrome to open as new tab

### DISCUSSION THREAD

**Discussion by LukaszP2** [linuxserver/Heimdall#1137] [Ideas]
It will be good when open new tab in chrome to view my page.

**Comment by SinisterSpatula** [comment]
Use a browser based on chrome if you want that ability immediately.  Brave Browser supports it already, just set "New Tab Page opens: Homepage" and set your Homepage to your heimdall dashboard.  I think edge supports it too.  It's just google being google and forcing you to see their new tab page.
